web/wp-content/themes/borealis-2021/header.php

```diff
@@ -67,23 +64,34 @@
             class="<?php echo $no_header ? 'min-h-[125px]' : esc_attr('bg-cover bg-bottom min-h-[400px] md:min-h-[280px] flex flex-col justify-end') ?>"
             style="background-image: url(<?php echo $no_header ? '' : esc_attr($hero_image) ?> )"
         >
-            <nav id="main-navigation" class="fixed left-0 right-0 top-3">
-                <div class="<?php echo ($hasSubnav)? 'rounded-t-large': 'rounded-large';?> relative mt-4 top-2 pb-2 pt-4 nav-container <?php echo (is_home() || is_front_page())? 'bg-transparent': 'bg-primary-navy-400'; ?>">
-                    <div class="relative flex container">
-                        <div class="pt-2 logo h-fit">
+            <nav id="main-navigation" class="fixed z-10 left-0 right-0 top-3">
+                <div class="<?php echo ($has_subnav)? 'rounded-b-large md:rounded-b-none rounded-t-large': 'rounded-large';?> relative mt-4 top-2 py-4 nav-container <?php echo (is_home() || is_front_page())? 'bg-transparent': 'bg-primary-navy-400'; ?>">
+                    <div class="flex md:flex-row flex-col items-center px-4 md:px-5 lg:px-0 lg:container">
+                        <div class="logo shrink-0 h-fit flex md:w-auto w-full justify-between align-center">
                             <a href="<?php echo get_home_url(); ?>">
                                 <img src="<?php echo $logoUrl ?>" alt="">
                             </a>
+                            <button role="button" class="text-shade-white-400 icon--lg md:hidden menu-toggle" data-toggle="main-nav-items" aria-expanded="false" aria-label="Open Main Menu">
+                                <?php echo pg_render_icon('hamburger') ?>
+                            </button>
                         </div>
-                        <div class="nav-items grow flex text-shade-white-400 py-4">
-                            <?php 
-                                $Menu->generate_menu('navigation-main', 'multi-level');
-                            ?>
+                        <div id="main-nav-items" class="nav-items grow flex fixed md:static inset-0 -left-full w-screen h-screen md:h-auto z-50 md:justify-end items-center md:text-shade-white-400 opacity-0 md:opacity-100 transition-opacity transition-left duration-500 bg-shade-black-30 md:bg-transparent max-h-screen">
+                            <div class="w-3/4 md:w-auto bg-shade-white-400 md:bg-transparent h-full md:flex-row flex-col flex pb-14 md:pb-0 md:items-center overflow-scroll">
```

Comment:
the overflow-scroll is messing with the desktop styling but other than that, solid gold!